### PR TITLE
(blog) Announcing release of CCM 0.4.0

### PR DESCRIPTION
**Pull request by pauby**
Blog post announcing the exciting release of Chocolatey Central Management 0.4.0 which includes:

* Deployment Scheduling
* Support for Semi-Connected Environments
* API

@designxsteph Can you please check the `` that I'm using is correct - I'm taking this from a previous blog post and I've noticed `w-100` and `w-60` being used. Just want to make sure it's using the right ones.

@ferventcoder The Zoom webinar page needs updated before this is pushed.

### CODE REVIEW

**Comment by designxsteph** [comment]
@pauby The `w-100' class needs to be removed from all images in this post. That class will force an object to be 100% of the width of its container. In this case it's making the image bigger than it actually is, which is resulting in a blurry image.

File: chocolatey/Website/Views/Blog/AnnouncingSchedulingAPISemiConnectedEnvironments.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+You can add a schedule to any deployment by selecting a start time. If you need to use an end time, for a maintenance window for example, you can add that as well!
+
+<div class="text-center"><img class="img-fluid border mb-3 w-100" src="/content/images/blog/CCM_Deployments_Edit_Schedule.gif" alt="Edit Deployment Schedule" title="Edit Deployment Schedule" /></div>
+
+Find out more about [working with Central Management Deployments](https://chocolatey.org/docs/central-management-deployments).
+
+### Support Semi-Connected Environments with Central Management Deployments
+With organizations allowing staff to work from home, we’ve been hearing from customers who want to use Central Management Deployments in environments where the endpoints are not always online.
+
+<div class="text-center"><img class="img-fluid border mb-3 w-100" src="/content/images/blog/CCM_Deployments_Edit_MachineContactTimeout.jpg" alt="Machine Contact Timeout" title="Machine Contact Timeout" /></div>
```
Comment (designxsteph) [inline chocolatey/Website/Views/Blog/AnnouncingSchedulingAPISemiConnectedEnvironments.md:33]: Remove `w-100`

File: chocolatey/Website/Views/Blog/AnnouncingSchedulingAPISemiConnectedEnvironments.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+To allow you to deploy to these environments, Central Management Deployments now allows you to set a Machine Contact Timeout to wait for a machine to be connected to the network before it is given a deployment step (and even make it wait indefinitely for those machines to check in).
+
+Find out more about [working with Central Management Deployments in a Semi-Connected Environment](https://chocolatey.org/docs/central-management-deployments#how-can-i-run-deployments-in-a-semi-connected-environment).
+
+### Automate Central Management with the new API
+<div class="text-center"><img class="img-fluid border mb-3 w-100" src="/content/images/blog/CCM_API.gif" alt="CCM API" title="CCM API" /></div>
```
Comment (designxsteph) [inline chocolatey/Website/Views/Blog/AnnouncingSchedulingAPISemiConnectedEnvironments.md:40]: Remove `w-100`

File: chocolatey/Website/Views/Blog/AnnouncingSchedulingAPISemiConnectedEnvironments.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+### What is Chocolatey Central Management Deployments
+If you’re not familiar with Chocolatey Central Management Deployments then our previous blog post announcing its release will help you understand what Deployments are, and how you can use them to easily manage your Windows endpoints. You can also find more information on our documentation pages.
+
+### Start Central Management Deployments On A Schedule Or During a Maintenance Window
+This is an exciting new feature to help you schedule deployments for a specific time period such as during a maintenance window, allowing you to manage your endpoints in line with your current procedures or workflows.
+
+You can add a schedule to any deployment by selecting a start time. If you need to use an end time, for a maintenance window for example, you can add that as well!
+
+<div class="text-center"><img class="img-fluid border mb-3 w-100" src="/content/images/blog/CCM_Deployments_Edit_Schedule.gif" alt="Edit Deployment Schedule" title="Edit Deployment Schedule" /></div>
```
Comment (designxsteph) [inline chocolatey/Website/Views/Blog/AnnouncingSchedulingAPISemiConnectedEnvironments.md:26]: Remove `w-100`

**Review by designxsteph — COMMENTED** [review]
@pauby The w-100 class needs to be removed from all images in this post. That class will force an object to be 100% of the width of its container. In this case it's making the image bigger than it actually is, which is resulting in a blurry image.

File: chocolatey/Website/Views/Blog/AnnouncingSchedulingAPISemiConnectedEnvironments.md (hunk 1 above)
Comment (pauby) [reply chocolatey/Website/Views/Blog/AnnouncingSchedulingAPISemiConnectedEnvironments.md]: Done.

File: chocolatey/Website/Views/Blog/AnnouncingSchedulingAPISemiConnectedEnvironments.md (hunk 3 above)
Comment (pauby) [reply chocolatey/Website/Views/Blog/AnnouncingSchedulingAPISemiConnectedEnvironments.md]: Done.

File: chocolatey/Website/Views/Blog/AnnouncingSchedulingAPISemiConnectedEnvironments.md (hunk 2 above)
Comment (pauby) [reply chocolatey/Website/Views/Blog/AnnouncingSchedulingAPISemiConnectedEnvironments.md]: Done.

**Review by designxsteph — APPROVED** [review]

**Review by ferventcoder — APPROVED** [review]

**Pull request merged**